assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -36,7 +37,9 @@ export default function SettingsForm() {
 	return (
 		<div className="googlesitekit-analytics-settings-fields">
 			<StoreErrorNotice moduleSlug="analytics" storeName={ STORE_NAME } />
+
 			<ExistingTagNotice />
+			<ExistingGTMPropertyNotice />
```

Comment:
See above, I think we should wrap this in a clause `if ( ! existingTag )`.

Reply:
Updated.